lib/dentaku/ast/functions/default.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../function'
+
+module Dentaku
+  module AST
+    class Default < Function
+      def self.type_syntax
+        "default(%a, %a) = %a"
```

Comment:
One issue is that we don't know if the first arg is truly just an identifier. It map be possible to hack a special case for this into the tokenizer and throw an error.

Reply:
Do you know if there's a way we can be more confident about this?

Reply:
@rtlong not without special casing this in the tokenizer or parser. It's not actually that big of a deal, using a literal as the the first arg still works e.g. `default(10, 100)` => `10`